src/domain/challenge/challenge/challenge.interface.ts

```diff
@@ -1,10 +1,13 @@
 import { IOpportunity } from '@domain/collaboration/opportunity/opportunity.interface';
 import { Field, ObjectType } from '@nestjs/graphql';
-import { IBaseChallenge } from '@domain/challenge/base-challenge/base.challenge.interface';
 import { IInnovationFlow } from '../innovation-flow/innovation.flow.interface';
 import { IStorageAggregator } from '@domain/storage/storage-aggregator/storage.aggregator.interface';
-@ObjectType('Challenge')
-export abstract class IChallenge extends IBaseChallenge {
+import { IJourney } from '../base-challenge/journey.interface';
+import { IBaseChallenge } from '../base-challenge/base.challenge.interface';
+@ObjectType('Challenge', {
+  implements: () => [IJourney],
+})
+export abstract class IChallenge extends IBaseChallenge implements IJourney {
```

Comment:
`IBaseChallenge` and `IJourney` are 100% the same. 
I think we should discuss the structure further.